src/main/java/org/gridsuite/modification/server/repositories/ModificationRepository.java

```diff
@@ -0,0 +1,26 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.repositories;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.gridsuite.modification.server.entities.ModificationEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@Repository
+public interface ModificationRepository extends JpaRepository<ModificationEntity, UUID> {
+    @Query(value = "SELECT *, 0 AS clazz_ FROM modification WHERE group_uuid = ?1", nativeQuery = true)
```

Comment:
can we use @DiscriminatorColumn and @DiscriminatorValue to avoid the magic of "0" and "clazz_" ?

Reply:
Only for InheritanceType.SINGLE_TABLE ?